src/components/book-content-mixin.cjsx

```diff
@@ -29,6 +33,36 @@ module.exports =
       else
         img.onload = sizeImage
 
+  getMediaTag: (media) ->
+    # form media tag text based on tag name or data-type
+    tag = media.tagName
+    tag = media.dataset.type if media.dataset.type?
+
+    # capitalize
+    # TODO find one place for all such string formating helpers to go
+    tag = tag.charAt(0).toUpperCase() + tag.substring(1).toLowerCase()
+
+  isMediaLink: (link) ->
+    link.innerText is '[link]' and link.hash.length > 0 and link.hash.search('/') is -1
```

Comment:
oh wait, for href, they all will have the UUID, we will have to do more detailed matching for the different cases, working on that now